docs/whatsnew/1.0.0/1_0_0.rst

```diff
@@ -0,0 +1,81 @@
+
+Creating kymographs from camera recordings
+------------------------------------------
+
+.. image:: kymo_to_stack.png
+
+A new feature introduced in `v1.0.0` is the ability to convert camera recordings to kymographs.
+
+First define a tether using :meth:`~lumicks.pylake.ImageStack.define_tether` which rotates the camera images such that the tether is aligned with the horizontal axis. Then, simply call :meth:`~lumicks.pylake.ImageStack.to_kymo`  to sum a region around the tether and construct a :class:`~lumicks.pylake.kymo.Kymo`.
+
+To read more about this feature, please refer to the :ref:`tutorial<kymo_from_image>`.
+
+Bias correction
+---------------
+
+When tracking kymographs, the default refinement algorithm used to achieve sub-pixel localization accuracy is based on centroid refinement.
+Centroid refinement can suffer from bias (tending towards the pixel center) when there is considerable background in the image.
+Pylake `1.0.0` removes this bias using the method presented in :cite:`berglund2008fast`.
+For best results, we still recommend using :ref:`gaussian refinement<kymotracker_gaussian_refinement>` after tracking.
+
+.. figure:: bias_correction.png
+
+    Localization in the presence of high background.
+    On the x-axis is the true position of a fluorescent spot, while on the y-axis, we see its inferred location.
+
+    From left to right: regular centroid refinement, bias-corrected refinement and Gaussian refinement.
+    The step-like progression in the centroid case is due to the bias induced by the background intensity.
+    These steps originate from a bias towards the center of each pixel.
+    Note how the bias corrected variant more closely agrees with the Gaussian refinement result.
+
+    From top to bottom: The top row shows localization performed with a small window, while the bottom row shows localization performed with a small window.
+    For Gaussian refinement, a larger window is beneficial (as it uses more data and models the background), while for centroid refinement it can lead to a larger estimation variance because of the inclusion of more noise.
+
+Scalebars
+---------
+
+Sometimes, it can be nice to have scale bars.
+With Pylake `v1.0.0` you can add them to just about any image plot by simply including a `scale_bar=lk.ScaleBar()` argument to `.plot()` or `.export_video()`.
+See :class:`~lumicks.pylake.ScaleBar` for more information.
+
+.. figure:: scalebar.gif
+
+    Movie exported from Pylake with `scan.export_video("rgb", "scan_stack.gif", scale_bar=lk.ScaleBar()))`.
+
+Calibrated Images
+-----------------
+
+Camera images now show the image in microns rather than pixels.
+
+.. figure:: calibrated.png
+
+Other changes
+-------------
+
+In addition to these functionalities, we implemented various other bug-fixes and improvements.
+Since this is a major release, it also includes some breaking changes.
```

Comment:
put the more impactful stuff on the top:
~~~suggestion
Since this is a major release, it also includes some breaking changes.
As always, we implemented various bug-fixes and other improvements.
~~~

Reply:
You reviewed an older version, which didn't have the bit about `semver` yet. How about:
```
Since this is a major release, it also includes some breaking changes.
Note that we adhere to `semantic versioning <https://semver.org/>`_, meaning that we increment the major version number to indicate that there are breaking changes.

As always, we implemented various other bug-fixes and improvements.
```

Reply:
Sounds good to me, although the "also" and the "some" bother me a bit. But when I remove them it starts to sound off.

Reply:
Removing "also"  and "some". :)